api/graphite.go

```diff
@@ -101,63 +209,62 @@ func (s *Server) renderMetrics(ctx *middleware.Context, request models.GraphiteR
 	}
 
 	reqs := make([]models.Req, 0)
-	parsedTargets := make(map[string]string)
+
+	// consolidatorForPattern[<pattern>]<consolidateBy>
+	consolidatorForPattern := make(map[string]string)
```

Comment:
I am not sure we should fix this in this PR.  It is going to require a lot of refactoring of how we parse targets.  I think it is best if we address this after this PR has been merged.